nix/machines/signs/microvm-config.nix

```diff
@@ -0,0 +1,20 @@
+{ config, ... }:
+
+{
+  microvm.qemu.serialConsole = false;
+  microvm.qemu.extraArgs = [
+    "-serial" "pty"
+  ];
+
+  microvm.vcpu = 2;
+  microvm.mem = 4096;
```

Comment:
Very generous! wow.

Reply:
idk we got lots of ram now! hard to be stingy